client_promise/dto/promise.go

```diff
@@ -0,0 +1,8 @@
+package dto
+
+type ClientPromise struct {
```

Comment:
- Create structure `type Signature string`
- Add field `issuerSignature *Signature`, test if is `nil` by default
- Add field `BenefiterSignature` *Signature`, test if is `nil` by default